README.md

```diff
@@ -113,9 +113,15 @@ Run `make servewallet` to compile the code and run `servewallet`. `servewallet`
 serves the HTTP API. Changes to the backend code are *not* automatically detected, so you need to
 restart the server after changes.
 
-#### Update go dependencies
+#### Go dependencies
 
-Run `dep ensure` to update dependencies.
+Go dependencies are managed by `go mod`, and vendored using `make go-vendor`. The deps are vendored
+so that
+
+- offline builds work
+- dependency diffs are easier to inspect
+- less reliance on remote systems
+- because `gomobile bind` does not support Go modules yet
```

Comment:
Would be nice to link to their issue on github or elsewhere imho, if there is one.

Reply:
None seems to match our issue exactly, but this indicates that gomobile is not ready yet for go modules: https://github.com/golang/go/issues/27234